core/node/fee_model/src/l1_gas_price/gas_adjuster/mod.rs

```diff
@@ -200,6 +200,12 @@ impl GasAdjuster {
             PubdataSendingMode::Calldata => {
                 self.estimate_effective_gas_price() * self.pubdata_byte_gas()
             }
+            PubdataSendingMode::Custom => {
+                // TODO: at this point we are assuming that the operators are subsidizing the cost
```

Comment:
consider adding a github issue and referencing it here (instead of 'plain TODO')

Reply:
added